integrationtests/test_i18n.py

```diff
@@ -20,3 +53,88 @@ def test_js_english_interpolation(self):
         # Wait for page to load
         b.assert_element('input[name="name"][value="Untitled Workflow"]', wait=True)
         b.assert_element(".attribution .metadata", text="by Jane Doe")
+
+
+class TestLocaleSwitch(LocaleSwitcherIntegrationTest):
+    def setUp(self):
+        super().setUp()
+        self.user = self.account_admin.create_user(
+            "user@example.org", first_name="Jane", last_name="Doe"
+        )
+
+    def test_language_switch_in_django_templates(self):
+        # You can change from "en" to "el" and back and the UI will be translated
+        b = self.browser
+        b.visit("/")
+        b.assert_element('input[name="password"][placeholder="Password"]', wait=True)
+        b.assert_element('button[type="submit"]', text="Sign In")
+        self.switch_locale_django("el")
+        b.assert_element('input[name="password"][placeholder="Συνθηματικό"]', wait=True)
+        b.assert_element('button[type="submit"]', text="Σύνδεση")
+        self.switch_locale_django("en")
```

Comment:
Before switching locales, navigate to another page (`b.click("[Greek for 'Sign Up']")` springs to mind) to check that the chosen language persists to the next page.

(Don't refresh the page. Selenium is prone to races; after refreshing the page, your next `b.assert_element()` may check the _previous_ page or it may check the _next_ page. If the assertion happens to query the _previous_ page, the test isn't actually testing anything. You need to test for an element that _isn't_ on the previous page to ensure the test is useful.)